server/internal/config/config.go

```diff
@@ -155,7 +158,7 @@ func (o *OIDC) validate() error {
 	if o.ClientID == "" {
 		return fmt.Errorf("clientId must be set")
 	}
-	if o.ClientSecret == "" {
+	if !strings.Contains(o.IssuerURL, "cloudnatix.com") && o.ClientSecret == "" {
```

Comment:
I think it's ok to simply make `ClientSecret` optional to avoid hard-coding `cloudnatix.com`.